scripts/newinstall.sh

```diff
@@ -282,11 +306,18 @@ set -e
 ##########	Download optional component (python, git, ...)
 
 if true; then
-	if [[ $WITH_MINICONDA2 = 1 ]]; then
```

Comment:
This script is actually documented to be a bash script and not a sh script (see the installation instructions: https://pipelines.lsst.io/install/newinstall.html ) so I don't think there is a problem.